### ISSUE TITLE
certs:add loses previously configured https proxy map

### ISSUE THREAD

**Issue by IlyaSemenov** [dokku/dokku#2340]
Running `dokku certs:add` loses the previously configured https proxy map.

Consider the following output (running with official `dokku_client.sh` at 714da78df6e9bf175c2f7e4afc2b9588fa4adca2):

```console
semenov@daedra ~/work/myapp (dokku) $ dokku config|grep PORT
DOKKU_DOCKERFILE_PORTS: 80
DOKKU_PROXY_PORT_MAP:   http:80:80 https:443:80
semenov@daedra ~/work/myapp (dokku) $ tar c ssl/myapp.crt ssl/myapp.key | dokku certs:add
ssl/myapp.crt
ssl/myapp.key
-----> Unsetting myapp
-----> Unsetting DOKKU_PROXY_PORT_MAP
-----> Setting config vars
       DOKKU_PROXY_PORT_MAP: http:80:80
-----> Configuring *.myapp.co...(using built-in template)
-----> Configuring myapp.co...(using built-in template)
-----> Creating https nginx.conf
-----> Running nginx-pre-reload
       Reloading nginx
semenov@daedra ~/work/myapp (dokku) $ dokku config|grep PORT
DOKKU_DOCKERFILE_PORTS: 80
DOKKU_PROXY_PORT_MAP:   http:80:80
```

Output of the following commands (`uname -a`, etc.): please see the gist https://gist.github.com/IlyaSemenov/a2536e140514725b8fb5fea50072f405

Environment details (AWS, VirtualBox, physical, etc.): AWS, Ubuntu 16.04

How was dokku installed?: `curl https://raw.githubusercontent.com/dokku/dokku/v0.6.5/bootstrap.sh | sudo DOKKU_TAG=v0.6.5 bash`

Steps to Reproduce:
1. Publish a Docker app that EXPOSEs a single port 80
2. Add a SSL certificate with `dokku certs:add`
3. Add a proxy map for HTTPS (`dokku proxy:ports-add https:443:80`)
4. Ensure that the site opens at https://myapp.dokku.host just fine
5. Ensure that `dokku config|grep PORT` shows the same as in the output above
6. Run `dokku certs:add` again

Actual Results:
The proxy map for `https:443:80` disappears.

Expected Results:
The cert is updated, the proxy map doesn't change.

**Comment by josegonzalez** [comment]
What version of dokku are you running?

**Comment by IlyaSemenov** [comment]
@josegonzalez this is dokku 0.6.5, the full output of all diagnostic commands is in the Gist that I've provided: https://gist.github.com/IlyaSemenov/a2536e140514725b8fb5fea50072f405#file-gistfile1-txt-L73

**Comment by IlyaSemenov** [comment]
My bad, I didn't realise that 0.7.0 was out and that the problem was resolved by #2317
